qemu/tests/block_commit_reboot.py

```diff
@@ -0,0 +1,46 @@
+import time
+import random
+from qemu.tests import blk_commit
+
+
+class BlockCommitReboot(blk_commit.BlockCommit):
+
+    def reboot(self):
+        """
+        Reset guest with system_reset;
+        """
+        params = self.parser_test_args()
+        method = params.get("reboot_method", "system_reset")
+        return super(BlockCommitReboot, self).reboot(method=method, boot_check=False)
```

Comment:
i have checked `block_copy.BlockCopy` in `reboot` method, if `boot_check=False`, no matter what you give to `method`, it will always use `system_reset`.
Do you have others way to reboot the guest? if not, I don't thunk you need to add `reboot_method = "system_reset"` in the cfg, and use it here.

Reply:
@vivianQizhu  any other reboot scenario? what do you think?

Reply:
@hereischen Both ok to me, there won't be other reboot scenario at present, if so, this class can be dropped. The step sleep is needed since it can give random time for commit in different reboot stage, but this step can be moved to run() function directly after where action_when_start() called.

Reply:
No, if you drop this class, you will use `reboot(method="shell", boot_check=True)`.
And in the current implementation, there is no way you can change this to what you get from `BlockCommitReboot`. 
So, let's keep it for it explicitly shows using ‘system_reset’ to reboot.